contrib/google.golang.org/grpc/client.go

```diff
@@ -136,16 +147,21 @@ func UnaryClientInterceptor(opts ...Option) grpc.UnaryClientInterceptor {
 // doClientRequest starts a new span and invokes the handler with the new context
 // and options. The span should be finished by the caller.
 func doClientRequest(
-	ctx context.Context, cfg *config, method string, opts []grpc.CallOption,
+	ctx context.Context, cfg *config, method string, methodKind string, opts []grpc.CallOption,
 	handler func(ctx context.Context, opts []grpc.CallOption) error,
 ) (ddtrace.Span, error) {
+	var spanOpts []ddtrace.StartSpanOption
+	if methodKind != "" {
```

Comment:
Given that you're right about `startSpanFromContext` in that it's used in other places too where `kind` doesn't make sense, I think I liked your previous version more...

Reply:
I'll put `startSpanFromContext` back how it was. `doClientRequest` looks better this way though, I think.

Reply:
Sure. Feel free to keep it like that.